README.md

```diff
@@ -3,6 +3,30 @@
 `limitd-redis` is client for limits on top of `redis` using [Token Buckets](https://en.wikipedia.org/wiki/Token_bucket).
 It's a fork from [LimitDB](https://github.com/limitd/limitdb).
 
+## Table of Contents
```

Comment:
this is great, is it autogenerated? If so, do we have docs to regen it?

Reply:
Copilot :D